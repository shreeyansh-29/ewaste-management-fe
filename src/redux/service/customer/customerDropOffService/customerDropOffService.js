import {CUSTOMER_DROPOFF} from "../../../../container/constant/constant";
import api from "../../../../core/utilities/httpProvider";

export const customerDropOffService = () => {
  return api.post(CUSTOMER_DROPOFF);
};
