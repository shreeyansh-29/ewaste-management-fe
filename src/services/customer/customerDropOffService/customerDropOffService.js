import api from "../../../core/utilities/httpProvider";

export const customerDropOffService = (data) => {
  return api.get(
    `http://localhost:8083/customer/request/dropOff/viewCollectors?category=${data.payload}`
  );
};
