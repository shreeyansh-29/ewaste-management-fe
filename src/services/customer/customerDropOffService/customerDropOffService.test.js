/* eslint-disable no-undef */

import {customerDropOffService} from "./customerDropOffService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing customerDropOffService", () => {
  const data = {payload: {}};
  const Service = customerDropOffService;
  const wrapper = shallow(<Service {...data} />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
