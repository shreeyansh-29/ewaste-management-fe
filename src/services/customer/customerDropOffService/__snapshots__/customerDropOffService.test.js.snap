// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing customerDropOffService 1`] = `<Component />`;
